cosmos/collection.go

```diff
@@ -185,3 +185,20 @@ func (c Collection) ExecuteSproc(sprocName string, partitionKeyValue interface{}
 	return c.Client.ExecuteStoredProcedure(
 		c.GetContext(), c.DbName, c.Name, sprocName, opts, ret, args...)
 }
+
+func (c Collection) ReadFeed(etag, partitionKeyRangeId string, maxItems int, documents interface{}) (cosmosapi.ListDocumentsResponse, error) {
```

Comment:
Yes, the user needs to maintain which partition key ranges are "live" and the cursors/etags for incrementally reading them. I have no plans to do more work on the library currently, as this fullfills the use case I'm working with. If my use case needs better horizontal scaling at some point it might be convenient to add support to this library.